doc/snippets/Microsoft.Data/SqlDbTypeExtensions.xml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0"?>
+<docs>
+  <members name="SqlDbTypeExtensions">
+    <SqlDbTypeExtensions>
+      <summary>Extensions for SqlDbType.</summary>
+      <remarks>
+        <format type="text/markdown">
+          <![CDATA[  
+
+## Remarks  
+The SqlDbTypeExtensions class provides new SqlDbType enums. These are meant to be used for applications which cannot leverage the enums in SqlDbType, which are available in newer .NET runtime versions.
```

Comment:
When mentioning types in remarks, use `xref` instead.
i.e.. `<xref:Microsoft.Data.SqlDbTypeExtensions>` and `<xref:System.Data.SqlDbType>`

Reply:
Got it. Thanks